velox/expression/tests/utils/ArgumentTypeFuzzer.cpp

```diff
@@ -114,15 +208,19 @@ TypePtr ArgumentTypeFuzzer::fuzzReturnType() {
       "Only fuzzing uninitialized return type is allowed.");
 
   determineUnboundedTypeVariables();
-  if (signature_.returnType().baseName() == "any") {
+  determineUnboundedIntegerVariables(signature_.returnType());
+
+  const auto& returnType = signature_.returnType();
+
+  if (returnType.baseName() == "any") {
     returnType_ = randType();
-    return returnType_;
   } else {
     returnType_ = exec::SignatureBinder::tryResolveType(
```

Comment:
just curious, does `tryResolveType` apply the integer constraints when it generates the return type?

Reply:
@bikramSingh91 Yes.